client/state/inline-help/actions.js

```diff
@@ -36,6 +46,18 @@ export function setInlineHelpSearchQuery( searchQuery = '' ) {
 	};
 }
 
+/**
+ * Map the collection, populating each result object
+ * with the given support type value.
+ *
+ * @param {Array}   collection   - collection to populate.
+ * @param {string}  support_type - Support type to add to each result item.
+ * @returns {Array}                Populated collection.
+ */
+function mapWithSupportTypeProp( collection, support_type ) {
+	return map( collection, ( item ) => ( { ...item, support_type } ) );
```

Comment:
Can we consider `collection.map( ( item ) => ( { ...item, support_type } ) )` and not import `map` from `loadash`?

Reply:
yes! tackled [here](b6d9ae9b06a26ef5d189c529ddbb810d3f2c8895). Thanks for your suggestion.